packages/angular-query-experimental/src/create-base-query.ts

```diff
@@ -92,10 +92,12 @@ export function createBaseQuery<
         })
       })
       onCleanup(() => {
-        resultFromSubscriberSignal.set(null)
+        ngZone.run(() => resultFromSubscriberSignal.set(null))
       })
     },
     {
+      // Set allowSignalWrites to support Angular < v19
+      allowSignalWrites: true,
```

Comment:
@arnoud-dv 
This will log a warning on Angular `19`. You should check Angular version before setting this config.

Reply:
Didn't expect a warning message for a noop 🤦‍♂️

Thanks for letting me know.